index.js

```diff
@@ -2613,6 +2613,42 @@
   }
   S.ifElse = def('ifElse', {}, [Pred(a), Fn(a, b), Fn(a, b), a, b], ifElse);
 
+  //# when :: (a -> Boolean) -> (a -> b) -> a -> b
+  //.
+  //. Takes a unary predicate, a unary function, and a value of any type, and
+  //. returns the result of applying the function to the value if the value
+  //. satisfies the predicate; the value otherwise.
+  //.
+  //. ```javascript
+  //. > S.when(x => x < 0, Math.abs, -1)
+  //. 1
+  //.
+  //. > S.when(x => x < 0, Math.abs, 4)
+  //. 4
+  //. ```
```

Comment:
I was just letting you know that you can sleep peacefully knowing that the examples in the readme and on the website are always correct. :)